ababot/bot/src/types/stonk.rs

```diff
@@ -0,0 +1,28 @@
+use serde::{Deserialize, Serialize};
```

Comment:
As mentioned above, might be better to have `commands/[command name]/[command files]` instead of having files for a command spread all over the project

Reply:
Not in the scope of this PR